http-clients/src/main/java/com/palantir/remoting/http/UserAgents.java

```diff
@@ -30,13 +28,16 @@ public static String getUserAgent(String userAgent, String version) {
 
     /**
      * Constructs a user agent from the {@link java.util.jar.Attributes.Name#IMPLEMENTATION_TITLE} and {@link
-     * java.util.jar.Attributes.Name#IMPLEMENTATION_VERSION} of the package of the provided class. The default value for
-     * both properties is "{@value DEFAULT_VALUE}".
+     * java.util.jar.Attributes.Name#IMPLEMENTATION_VERSION} of the package of the provided class.
```

Comment:
can you add a little hint on how to get the correct information into that package?
"Typically, the title and version are extracted from {@code MANIFEST.MF} entries of the Jar package containing the given class."